src/Guide/Config.hs

```diff
@@ -60,6 +71,11 @@ instance FromJSON Config where
     _adminPassword <- o .:? "admin-password" .!= _adminPassword def
     _prerender     <- o .:? "prerender"      .!= _prerender def
     _discussLink   <- o .:? "discuss-link"   .!= _discussLink def
+    _portMain      <- o .:? "portMain"       .!= _portMain def
+    _portApi       <- o .:? "portApi"        .!= _portApi def
+    _portEkg       <- o .:? "portEkg"        .!= _portEkg def
+    _cors          <- o .:? "cors-switcher"  .!= _cors def
```

Comment:
Usually (in other software) such config fields don't have "switcher", "flag", etc in their names.

```suggestion
    _cors          <- o .:? "cors"           .!= _cors def
```